packages/react/src/components/RuleBuilder/RuleBuilderPropTypes.js

```diff
@@ -33,7 +33,12 @@ export const GroupLogicPropType = PropTypes.oneOf(['ALL', 'ANY']);
 export const RulesPropType = PropTypes.shape({
   id: PropTypes.string.isRequired,
   columnId: PropTypes.string.isRequired,
-  value: PropTypes.any.isRequired,
```

Comment:
There could be an argument for keeping any here and adding an exception, but I added all the various types used currently.